S17-procasync/windows-arg-quoting.t

```diff
@@ -0,0 +1,54 @@
+use v6;
+use Test;
+
+=begin pod
+
+Sub process arguments need to be quoted according to a specific schema
+to be understood by most processes on Windows.
+This quoting is tested here.
+
+See L<https://github.com/Raku/problem-solving/issues/20>
+
+=end pod
+
+plan 21;
+
+if !$*DISTRO.is-win {
```

Comment:
What about Java on windows? (it *did* work at one time)

Reply:
Good catch! Would you disable these tests altogether on JVM or only the failing ones?

Reply:
If NativeCall doesn't work on JVM it is probably a moot point. Do you happen to have a hunch on which ones would might fail on JVM?

Reply:
My guess is, that all the `test-run-verbatim` tests will fail because the JVM will just quote the things anyways (because the thing we call is not a .bat file).

Reply:
Now all of these tests are skipped on the JVM. I guess there isn't much point in testing JVMs internal quoting behavior.